environment/src/main/java/com/sequenceiq/environment/environment/flow/deletion/handler/RdbmsDeleteHandler.java

```diff
@@ -35,7 +35,7 @@ public void accept(Event<EnvironmentDeletionDto> environmentDtoEvent) {
                 .withResourceCrn(environmentDto.getResourceCrn())
                 .withResourceName(environmentDto.getName())
                 .withForceDelete(environmentDeletionDto.isForceDelete())
-                .withSelector(START_PUBLICKEY_DELETE_EVENT.selector())
+                .withSelector(START_ENCRYPTION_SET_FOR_RESOURCE_DELETE_EVENT.selector())
                 .build();
         try {
             eventSender().sendEvent(envDeleteEvent, environmentDtoEvent.getHeaders());
```

Comment:
@doktoric @biharitomi  What is the purpose of this state/handler? Nothing is done besides sending the progress event. I do not even get what RBDMS has to do with the env as it only exists at DL/DH level. 😃

Reply:
@lajosrodek if any experience provision any DB with redbeams we need to delete that.

Reply:
Oh, right, got it. That indeed makes sense then.